kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/src/test/java/org/kie/server/integrationtests/controller/KieControllerStartupIntegrationTest.java

```diff
@@ -220,10 +220,9 @@ public void testContainerDisposedAfterStartup() throws Exception {
         assertEquals(1, instanceList.size());
 
         // Turn kie server off, dispose container and start kie server again.
-        server.stopKieServer();
```

Comment:
This test verify that managed Kie server reflects container removal (made in controller) while being stopped during that removal. Please don't change order of the steps, that makes the tests useless.